### PR TITLE
Manual for foreman_chef 0.2 release

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: plugins/foreman_chef/0.2/index.md [hunk 1]
```diff
@@ -0,0 +1,510 @@
...
+if needed.
+
+SmartProxyChef plugin also requires Ruby 1.9 because of chef-api gem that
+we use to communicate with chef server. Smart proxy does not run in software
+collection so the support is limited to platforms where system ruby is 1.9+.
+Therefore you can use following (and newer) platforms RHEL 7, CentOS 7,
+Debian Wheezy, Ubuntu Trusty.
+
+Chef handler Foreman should run on Ruby 1.8+ so you can use it on most
+platforms, since 0.1.0 you can it works even on Windows.
```
Comment (mmoll) [inline plugins/foreman_chef/0.2/index.md]: either "you can can use it also on Windows" or "it works also on Windows"

**Comment by mmoll** [comment]
one little note, :+1: otherwise

**Comment by ares** [comment]
fixed, thanks for the review

**Comment by mmoll** [comment]
dík!

File: plugins/index.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ Plugins are tools to extend and modify the functionality of Foreman. The core of
 		</a>
 	</div>
 	<div class='col-md-3 center'>
-		<a href="plugins/foreman_chef/0.1" class="btn-doc btn">
+		<a href="plugins/foreman_chef/0.2" class="btn-doc btn">
```
Comment (domcleal) [inline plugins/index.md:22]: Other plugins have a landing page where users can select the version, did you want one for Chef here so users can still get to 0.1?

Comment (ares) [inline plugins/index.md:22]: hm, maybe I can just put links to older versions, the landing page doesn't look nice especially just for two versions

**Comment by ares** [comment]
I added a landing page and link to other versions, keeping link from plugin list to latest one, does it make more sense now?

**Comment by domcleal** [comment]
@ares sorry, I'm not sure what you mean.  @stbenjam added one in 3dece94, which looks good.

**Comment by ares** [comment]
argh I missed that, sorry for confusion

**Pull request merged**